src/GoodsList.tsx

```diff
@@ -8,7 +8,11 @@ type Props = {
 export const GoodsList: React.FC<Props> = ({ goods }) => (
```

Comment:
wrapping it with `memo()` would be a good optimization

Reply:
Done